modules/gapi/src/executor/gstreamingexecutor.cpp

```diff
@@ -1005,12 +1005,10 @@ cv::gimpl::GStreamingExecutor::GStreamingExecutor(std::unique_ptr<ade::Graph> &&
                     // In the current implementation, such islands
                     // _must_ start with copy
                     GAPI_Assert(isl->in_ops().size() == 1u);
-#if !defined(GAPI_STANDALONE)
```

Comment:
@dmatveev There's no need for that anymore, is there ?

Reply:
I agree